tools/SetupFlow/DevHome.SetupFlow/Services/SetupFlowOrchestrator.cs

```diff
@@ -171,12 +170,53 @@ private bool CanGoToPreviousPage()
         return HasPreviousPage && CurrentPageViewModel.CanGoToPreviousPage;
     }
 
+    // Teaching tip needs to be open and visible to show up.
+    [ObservableProperty]
+    private bool _isTeachingTipOpen;

```

Comment:
Not this PR, but we should consider moving all the bindable properties out of the service class and let the view model call into the service.

- https://github.com/microsoft/devhome/issues/1435